.azure_pipelines/ci-pipeline-makefile-nightly.yml

```diff
@@ -32,6 +32,7 @@ jobs:
         sudo apt-get install build-essential python3-setuptools python3-pip llvm-7 libmbedtls-dev docker-ce -y
         sudo chmod 666 /var/run/docker.sock
         sudo apt install python3-pip -y
+	sudo apt install lldb-10 -y
```

Comment:
yaml is space-sensitive, it seems to be failing out because it needs one more tab here

Reply:
Thanks! trying it out now.